src/_tokens/tokens.ts

```diff
@@ -1,6 +1,10 @@
 export const color = {
   black: '#000000',
+  blackTr10: 'rgba(0, 0, 0, 0.10)',
+  blackTr05: 'rgba(0, 0, 0, 0.05)',
   white: '#FFFFFF',
+  whiteTr50: 'rgba(255, 255, 255, 0.1)',
+  whiteTr05: 'rgba(255, 255, 255, 0.05)',
```

Comment:
Why these new values? We have a `SlateTr10` They are not part of our tokens in Figma. I'm not saying we shouldn't add those but I'm curious to know where this is coming from. If this is something we want to support, this should also be part of our Figma library.

Reply:
For shadows, but I'm happy to not create new tokens for that. Also, I didn't see a `SlateTr10`

Reply:
I would be happy to add them to Figma. I think these 2 new values could work.